python/sbp/client/loggers/device_iterator.py

```diff
@@ -0,0 +1,101 @@
+# Copyright (C) 2015 Swift Navigation Inc.
+# Contact: Mark Fine <mark@swiftnav.com>
+#
+# This source is subject to the license found in the file 'LICENSE' which must
+# be be distributed together with this source. All other rights reserved.
+#
+# THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+# EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+
+from base_logger import LogIterator
+from ...client.handler import Handler
+from ...table import dispatch
+import calendar
+import time
+from Queue import Queue, Empty
+
+
+
+class SbpQueue(object):
```

Comment:
Does this wrapper serve much purpose? Could you not just use the queue directly below?